farm/eval.py

```diff
@@ -37,7 +37,7 @@ def __init__(
         self.device = device
         self.report = report
 
-    def eval(self, model, return_preds_and_labels=False):
+    def eval(self, model, return_preds_and_labels=False, update_temp=False):
```

Comment:
just suggestion to use `update_confidence` here as `temp` sometime used for temporary :)

Reply:
Yes, agreed. I am thinking about a different naming. Maybe "calibrate_conf_scores"

Reply:
Yeah but yes instead of `conf` we can use full name `calibrate_confidence_scores`